Cubical/Algebra/Module/Instances/FinMatrix.agda

```diff
@@ -0,0 +1,39 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Module.Instances.FinMatrix where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.SIP
+
+open import Cubical.Functions.FunExtEquiv using (funExt₂)
+
+open import Cubical.Data.Nat using (ℕ)
+
+open import Cubical.Algebra.Matrix
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Module
+
+module _ {ℓ} (R : Ring ℓ) {m n : ℕ} where
+
+  open module R' = RingStr (str R) renaming (_+_ to _+r_; -_ to -r_)
+  open LeftModuleStr
+
+  FinVecLeftModule : LeftModule R ℓ
```

Comment:
Shouldn't it be called 'FinMatrixLeftModule' or something like that?

Reply:
I changed the name.